### PR TITLE
[2310-msi] Default stockId and from storeView.

### PR DESCRIPTION
**Pull request by dimasch**
\+ @pkarw

### CODE REVIEW

File: scripts/mage2vs.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ function getMagentoDefaultConfig(storeCode) {
     MAGENTO_ACCESS_TOKEN: apiConfig.accessToken,
     MAGENTO_ACCESS_TOKEN_SECRET: apiConfig.accessTokenSecret,
     MAGENTO_URL: apiConfig.url,
+    MAGENTO_MSI_STOCK_ID: 1,
```
Comment (pkarw) [inline scripts/mage2vs.js]: It should be rather: config.msi.defaultStockId

Comment (dimasch) [reply scripts/mage2vs.js]: @pkarw Fixed.

**Comment by pkarw** [comment]
@dimasch can You please just fix the failing CI - https://travis-ci.org/DivanteLtd/vue-storefront-api/jobs/526288380 ?

**Comment by dimasch** [comment]
Yes, ok, one minute.

**Comment by dimasch** [comment]
@pkarw Fixed eslint error.

**Pull request merged**